include/sys/zthr.h

```diff
@@ -28,7 +28,7 @@ extern zthr_t *zthr_create(const char *zthr_name,
     zthr_checkfunc_t checkfunc, zthr_func_t *func, void *arg);
 extern zthr_t *zthr_create_timer(const char *zthr_name,
     zthr_checkfunc_t *checkfunc, zthr_func_t *func, void *arg,
-	hrtime_t nano_wait);
+	hrtime_t nano_wait, pri_t pri);
```

Comment:
Why have you added it only to zthr_create_timer(), but not zthr_create()?  I'd prefer symmetric KPIs, and so I'd add it to both after arg.

Reply:
I thought about adding to both as well. I opted to update only zthr_create_timer() as arc_evict_zthr is only one needing a new priority and zthr_create() looks to be a simpler interface and should probably remain so.

I'm open to make the change if folks feel strong about adding priority parameter to both.

Reply:
Either way is fine with me, although I think I'd lean towards updating the `zthr_create()` interface as well.  Keeping them symmetric would be nice, but I think the bigger win is passing the priority helps self-document how important these threads are in the grand scheme of things.

Reply:
Yes, making thread priority clearer for those created from zthr_create() is a good point. I'll make the update.

I completed the testing with defsysclprio, i.e. nice=0, which has similar performance improvements with lower CPU consumption so the revision will use defsysclprio for arc_evict thread.